src/lib/pages/backoffice/Document/DocumentDetails/DocumentContent.js

```diff
@@ -26,6 +27,20 @@ export class DocumentContent extends Component {
           </Accordion.Content>
         ),
       },
+      {
+        key: 'document-borrowing-requests',
```

Comment:
I would maybe place it under "Physical items" section since ILLs are about external libraries to have grouping "local" loans and items and then "external". Not super strong opinion. I would send your screenshot to Lydia and ask her what feels more handy to her in terms of placing this section